defaultModel/src/main/resources/defaultModelConfig.conf

```diff
@@ -30,4 +30,6 @@
   componentsGroupMapping {
     hidden: null
   }
+
+  fetchKafkaTopicsWithoutSchema: true
```

Comment:
the default imo should be somewhere in code (like KafkaConfig.topicsExistenceValidationConfig) - afaik if we don't use defaultModel.jar in installation then it won't be applied